mempool/mempool.go

```diff
@@ -646,6 +650,10 @@ func (mp *TxPool) RemoveDoubleSpends(tx *dcrutil.Tx) {
 // This function MUST be called with the mempool lock held (for writes).
 func (mp *TxPool) addTransaction(utxoView *blockchain.UtxoViewpoint,
 	tx *dcrutil.Tx, txType stake.TxType, height int64, fee int64) {
+	if txType == stake.TxTypeSSGen {
+		log.Debug("Signalling Regen Template")
```

Comment:
I don't see much value in this debug statement.  If we're going to leave it in, it should at least provide some additional details.  I'd prefer to just nuke it though.

Reply:
Yeah, that was a local debug that slipped through. Will be removed.